app/models/user.rb

```diff
@@ -40,6 +40,8 @@ class User < ActiveRecord::Base
 
   devise :omniauthable
 
+  paginates_per 10 #MAUD
```

Comment:
I would not add the pagination configiration here because it adds an (in my opinion) unneeded coupling between the persistence and the presentation layer. The model shouldn't care how clients to it present it. I know kaminari supports it but I think it's a bad design decision on their side. I think there is a `per` scope you can call on the `page` scope (as called in the controller) but that's just serving from memory.